src/Model.elm

```diff
@@ -72,14 +68,22 @@ type alias Draft =
     }
 
 
+type alias NotificationAggregate =
```

Comment:
I debated that. I felt like this was something specific to Tooty, but we may want to move it into the Mastodon module indeed.